src/openrct2/world/QuarterTile.h

```diff
@@ -0,0 +1,42 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2024 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <cstdint>
+
+class QuarterTile
+{
+private:
+    uint8_t _val{ 0 };
+
+public:
+    constexpr QuarterTile(uint8_t tileQuarter, uint8_t zQuarter)
+        : _val(tileQuarter | (zQuarter << 4))
+    {
+    }
+
+    QuarterTile(uint8_t tileAndZQuarter)
+        : _val(tileAndZQuarter)
+    {
+    }
```

Comment:
I know you just moved them, but I don't get why one is `constepxr` and the other isn't 🤔 

Assuming they can both be `constexpr`, could they also use delegating constructors?
```c++
    constexpr QuarterTile(uint8_t tileAndZQuarter)
        : _val(tileAndZQuarter)
    {
    }
   constexpr QuarterTile(uint8_t tileQuarter, uint8_t zQuarter)
        : QuarterTile(tileQuarter | (zQuarter << 4))
    {
    }
```

Reply:
Was it not possible to make them delegating constructors too?